File: redact-dropwizard-example/src/main/java/pii/PiiBody.java
package pii;

import com.fasterxml.jackson.annotation.JsonProperty;

public class PiiBody {
    @JsonProperty
    Password password;
    @JsonProperty
    SSN ssn;
    @JsonProperty
    String name;
    @JsonProperty
    String redact;

    @Override
    public String toString() {
        return "PiiBody{" +
                "password=" + password +
                ", ssn=" + ssn +
                ", name='" + name + '\'' +
                '}';
    }
}

File: redact-dropwizard-kotlin-example/src/main/java/us/jsy/redact/pii/PiiBody.kt
package us.jsy.redact.pii

import com.fasterxml.jackson.annotation.JsonProperty

class PiiBody {
    @JsonProperty
    var password: Password? = null

    @JsonProperty
    var ssn: SSN? = null

    @JsonProperty
    var name: String? = null

    @JsonProperty
    var redact: String? = null
    override fun toString(): String {
        return "PiiBody{" +
                "password=" + password +
                ", ssn=" + ssn +
                ", name='" + name + '\'' +
                '}'
    }
}